frontend/src/pages/site/components/main/styles.module.scss

```diff
@@ -43,7 +43,7 @@
 }
 
 .main-picture {
-  padding: 70px 20px;
+  height: 750px;
```

Comment:
maybe it bad idea,but maybe use height: 100% for img ,because main container dont have static height

Reply:
good point, but is there any other way to reduce image size in main section?